website/docs/r/dnat.html.markdown

```diff
@@ -43,6 +44,7 @@ resource "vcd_dnat" "forIcmp" {
 The following arguments are supported:
 
 * `edge_gateway` - (Required) The name of the edge gateway on which to apply the DNAT
+* `network_name` - (Optional *v2.2+*) The name of the organization network on which to apply the DNAT. This will be required field in the next major version.
```

Comment:
Looks good. Only please add a semicolon after Optional and emphasis to the warning:

`network_name` - (Optional; *v2.2+*) The name of the organization network on which to apply the DNAT. *`network_name` will be required field in the next major version*

Reply:
fixed